WordPress/Classes/ViewRelated/Plans/PlanListViewController.swift

```diff
@@ -91,6 +91,45 @@ enum PlanListViewModel {
             )
         }
     }
+    
+    func tableFooterView() -> UIView? {
+        switch self {
+        case .Ready:
+            let title = footerTitle
+            let footerView = WPTableViewSectionHeaderFooterView(reuseIdentifier: "ToSFooterView", style: .Footer)
+        
+            footerView.attributedTitle = title
+            footerView.frame.size.height = WPTableViewSectionHeaderFooterView.heightForFooter(title.string, width: footerView.bounds.width)
+            
+            return footerView
+        default:
+            return nil
+        }
+    }
+    
+    private var footerTitle: NSAttributedString {
+        let plainTosText = NSLocalizedString("By checking out, you agree to our fascinating terms and conditions.",
+                                             comment: "TOS label when making a purchase");
+        let tosText = NSLocalizedString("fascinating terms and conditions",
+                                        comment: "'fascinating terms and conditions' should be the same  text that is at the end of 'TOS label when making a purchase'")
```

Comment:
This might be confusing for translators, as comments aren't always very apparent on translation tools. I'd rather have a single string with some sort of simple markup.

It's not very pretty, but the sign in refactor PR deals with a [similar case](https://github.com/wordpress-mobile/WordPress-iOS/pull/5126/files/ee52dee489c236d0c5d2bd12b2d5f5ef4e3edebc#diff-e1e87cff9d5d7809319e55fbdfee3755R80) by using HTML:

```swift
let tos = "By checking out, you agree to our <a>fascinating terms and conditions</a>."
let styledTos = "<style>body {font-family: sans-serif; font-size:14px; color: #ffffff; text-align:center;} a {text-decoration: underline; color: #0000FF;}</style>" + tos
let attributedTos = try! NSAttributedString(
    data: styledTos.dataUsingEncoding(NSUTF8StringEncoding)!,
    options: [NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType],
    documentAttributes: nil)
```